description.md

```diff
@@ -21,6 +21,47 @@ Each section details how a specific type of content is to be developed, stored a
 
 ### Tutorials
 
+Tutorials are pieces of content aimed at walking the participant through a succession of steps that are commonly used in practice.
+Tutorial steps are supposed to be carried out by participants individually.
+They can be used as part of a practical session, with assistance provided to further explaining certain items, or as unassisted self-study.
+At the end of the tutorial, participants will be able replicate the results, and, while doing so, gain the basic practical skills related to the topic.
+These skills are to be improved by subsequent practical activities, such as ["Drills"](#drills) or ["Projects"](#projects).
+
+When compared to drills or projects, tutorials tend to be less engaging for participants:
+participants are advised to follow a set of predefined steps, and there is no challenge in finding a solution.
+Because of that, in actual activities, such as ["Practical Sessions"](#practical-sessions), tutorials are to be blended with the more engaging content items such as ["Quizzes"](#quizzes) and ["Drills"](#drills).
```

Comment:
```suggestion
Because of that, in actual activities, such as ["Practical Sessions"](#practical-sessions), tutorials are to be blended with more engaging content items such as ["Quizzes"](#quizzes) and ["Drills"](#drills).
```